ergo-rest/src/api/node.rs

```diff
@@ -92,6 +92,6 @@ mod tests {
             timeout: Some(Duration::from_secs(5)),
         };
         let res = runtime_inner.block_on(async { get_info(node_conf).await.unwrap() });
-        assert_eq!(res.name, "ergo-mainnet-4.0.16.1");
```

Comment:
The node now replies with "ew" as its name. We can't anticipate any future changes here, so check that it's not empty.

Reply:
Thanks! My bad, for making the test so fragile. I was very happy it worked so I forgot to make a more sensible check. :)

Reply:
I know the feeling!